src/main/java/com/mojang/brigadier/arguments/StringArgumentType.java

```diff
@@ -87,7 +87,8 @@ private static String escape(final String input) {
     public enum StringType {
         SINGLE_WORD("word", "words_with_underscores"),
         QUOTABLE_PHRASE("\"quoted phrase\"", "word", "\"\""),
-        GREEDY_PHRASE("word", "words with spaces", "\"and symbols\""),;
+        GREEDY_PHRASE("word", "words with spaces", "\"and symbols\""),
+        TERM("predefined_tokens","red","green","blue"),;
```

Comment:
`predefined_token` instead of `predefined_tokens`

Reply:
applied.